autoload/go/lint.vim

```diff
@@ -95,6 +95,87 @@ function! go#lint#Gometa(bang, autosave, ...) abort
   endif
 endfunction
 
+function! go#lint#Golangci(bang, autosave, ...) abort
+  if a:0 == 0
+    let goargs = [expand('%:p:h')]
+  else
+    let goargs = a:000
+  endif
+
+  if empty(go#config#GolangciLintCommand())
+    let bin_path = go#path#CheckBinPath("golangci-lint")
+    if empty(bin_path)
+      return
+    endif
+
+    let cmd = [bin_path]
+    let cmd += ["run"]
+    let cmd += ["--print-issued-lines=false"]
+    let cmd += ["--fast"]
```

Comment:
this flag makes it so only the "fast" linters run, probably don't want to hardcode this